org.codeberg.DrRac27.Karoto.yml

```diff
@@ -0,0 +1,40 @@
+app-id: org.codeberg.DrRac27.Karoto
```

Comment:
Oh ok thanks, will change. Is this flathub specific or should I change this everywhere like in the AppStream because it is in some FreeDesktop specification for example?

Reply:
The goal is to have a developer controlled namespace, and that can be eventually reached as a web resource. The flatpak doc mentions it in the context of github and gitlab.

We enforce it on flathub (existing one don't get changed), but this totally in line with the xdg specification.

Reply:
yeah

Reply:
Do I have to create a new PR with a new branch name or is it ok like this?

Reply:
the branch name is 10000% irrelevant.